cloud-consumption-api/src/main/java/com/sequenceiq/consumption/api/v1/consumption/model/request/StorageConsumptionRequest.java

```diff
@@ -16,6 +17,14 @@ public class StorageConsumptionRequest extends ConsumptionBaseRequest {
     @ApiModelProperty(value = ConsumptionModelDescription.STORAGE_LOCATION, required = true)
     private String storageLocation;
 
+    @NotEmpty
+    @ApiModelProperty(value = ConsumptionModelDescription.CONSUMPTION_TYPE, required = true)
```

Comment:
could you please add the possible values to the swagger doc?

Reply:
Which file exactly?

Reply:
the annotation has a values, where you can define the possible values

Reply:
Done